src/services/microfrontend-service.ts

```diff
@@ -90,6 +90,33 @@ export class MicroFrontendService {
     return microfrontends.find(({ name }) => name === mfeName)
   }
 
+  private isConfigMfeSiblingWidget(mfe: MicroFrontend): boolean {
+    return (
+      'configMfe' in mfe &&
+      mfe.configMfe !== undefined &&
+      mfe.configMfe.slice(0, 9) !== 'internal:'
```

Comment:
Maybe we can extract `internal:` to a constant, shared also with `component-service.ts`.

Reply:
done